arlo-client/src/components/AuditFlow/BoardTable.tsx

```diff
@@ -121,7 +121,8 @@ const BoardTable: React.FC<IProps> = ({
     setCols(columnWidths())
   }, [ballots])
 
-  const roundComplete = ballots && ballots.every(b => b.status === 'AUDITED')
+  const roundComplete =
+    ballots.length && ballots.every(b => b.status === 'AUDITED')
```

Comment:
```suggestion
    ballots.some(b => b.status !== 'AUDITED')
```

Reply:
@eventualbuddha The `ballots.length` is necessary to test that it's not an empty array, in which case it's not complete -- it's not even started yet.

`ballots.some(b => b.status !== 'AUDITED')` would return `true` if any of the ballots are incomplete, right? What we want is to test that *none* of them are incomplete. All of them have to be audited for the round to be done, which is what this variable measures.

Reply:
`Array#some` will always return `false` for an empty array, so that covers the `ballots.length` part. And `ballots.every(b => b.status === 'AUDITED')` checks that _all_ ballots have status `AUDITED`. `ballots.some(b => b.status !== 'AUDITED')` checks that _any_ ballot has a status other than `AUDITED`. Those are equivalent statements, right?